### ISSUE TITLE
The usage of Log4j2 Strings Util class in the code makes users force dependency on log4j2.

### ISSUE THREAD

**Issue by aofall** [sofastack/sofa-registry#345]
## In what area(s)?

 /area test-and-release

https://github.com/sofastack/sofa-registry/blob/3d8297aae0017ed222e71add10f11bd2deeb23ec/server/server/session/src/main/java/com/alipay/sofa/registry/server/session/wrapper/BlacklistWrapperInterceptor.java#L31

https://github.com/sofastack/sofa-registry/blob/3d8297aae0017ed222e71add10f11bd2deeb23ec/server/server/integration/src/main/java/com/alipay/sofa/registry/server/integration/RegistryApplication.java#L45

## Describe the feature

Please use the commons-lang3 library or maintain self StringUtils class in the project.

**Comment by nocvalight** [comment]
I agree with you, we can use commons-lang3 StringUtils to replaces Strings in log4j-api, and upgrade commons-lang to commons-lang3 in the project.
Can you help to commit a pr to do this?

**Comment by aofall** [comment]
Will it release 

> I agree with you, we can use commons-lang3 StringUtils to replaces Strings in log4j-api, and upgrade commons-lang to commons-lang3 in the project. Can you help to commit a pr to do this?

Done. PTAL

By the way, Is there a plan to publish new version to the Maven repository recently? The [dubbo-spi-extension](https://github.com/apache/dubbo-spi-extensions) repository have some modules about [sofa-registry](https://github.com/sofastack/sofa-registry) and we has been doing version upgrading and writing unit tests. The is some problem encountered this issue when writing unit tests.

**Comment by nocvalight** [comment]
> > I agree with you, we can use commons-lang3 StringUtils to replaces Strings in log4j-api, and upgrade commons-lang to commons-lang3 in the project. Can you help to commit a pr to do this?
> 
> Done. PTAL
> 
> By the way, Is there a plan to publish new version to the Maven repository recently? The [dubbo-spi-extension](https://github.com/apache/dubbo-spi-extensions) repository have some modules about [sofa-registry](https://github.com/sofastack/sofa-registry) and we has been doing version upgrading and writing unit tests. The is some problem encountered this issue when writing unit tests.

why use common-lang will cause problem when writing unit tests, what are the problems?

**Comment by aofall** [comment]
> why use common-lang will cause problem when writing unit tests, what are the problems?

The main issue is the `Strings` reference to log4j-api. In the code left by previous developers, log4j-api was excluded and logback was introduced as an additional dependency, It will cause `java.lang.ClassNotFoundException` when used the `TestRegistryMain.startRegistry()`. 

https://github.com/apache/dubbo-spi-extensions/blob/9e99495f897493900f9b6ee2d961cf264484fd29/dubbo-registry-extensions/dubbo-registry-sofa/pom.xml#L101

I can certainly remove the exclusion statements, but I would recommend not force users dependency on log4j2. I just upgraded commons-lang to commons-lang3 based on this suggestion. 

> and upgrade commons-lang to commons-lang3 in the project.

However, some modules in the project still has the commons-lang library such as `registry-server-data`, which is transitive dependency by `jraft-core`.

If upgrade commons-lang to commons-lang3 is not necessary, I will close this PR and recreate another pr.
